spec/findbugs_translate_checkstyle_format/translate_spec.rb

```diff
@@ -143,6 +143,17 @@
         expect(FindbugsTranslateCheckstyleFormat::Translate.fqcn_to_path('com.example.Test$1', xml)).to eq '/test/com/example/Test.java'
       end
 
+      it 'inner class' do
+        xml = {
+          'BugCollection' => {
+            'Project' => {
+              'SrcDir' => '/test/com/example/Test.java'
+            }
+          }
+        }
+        expect(FindbugsTranslateCheckstyleFormat::Translate.fqcn_to_path('com.example.Test$TestClass', xml)).to eq '/test/com/example/Test.java'
```

Comment:
__[rubocop]__
* Line is too long. [144/120] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)